test/WebSites/FormatFilterWebSite/Controllers/ProducesOverrideClass.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace FormatFilterWebSite
+{
+    [Produces("application/custom_ProducesOverrideController")]
+    public class ProducesOverrideController : ProducesBaseController
```

Comment:
When we talk about overriding with filters, we're generally referring to scoping.

```
[Produces(...)]
public class C
{
    [Produces(...)]
    public void A() { }
}
```